### DISCUSSION TITLE
Fixed regression with the -n parameter.

### DISCUSSION THREAD

**Discussion by Shabbatt** [Chia-Network/bladebit#225] [General]
"Fixed regression with the -n parameter."
does not work.
Surprisingly, if you specify this parameter once, it stops running without it.
Check, please.

**Comment by Shabbatt** [comment]
once, run with the -n option
a window opened. stood at the start of the memory.
then it disappeared.
after that, even now, I remove this parameter, the plotting does not start.

**Comment by Shabbatt** [comment]
this is how it worked.
is no longer working now. specified -n once, did not start.
removed this parameter and now it does not start at all.

what am I doing wrong?
help me please.

timeout 30
"@echo off

set plots=5
set threads=47
set buckets=64
set tmpdir2=I:\
set finaldir=H:\ChiaNew\
set poolkey=****************
set farmerkey=**************
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
echo %date% - %time%
echo -------------------------
bladebit.exe -t %threads% -f %farmerkey% -c %poolkey% diskplot -b %buckets% -a --cache 99G -t1 %tmpdir2% %finaldir% 
echo -------------------------
echo %date% - %time%
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
pause"

**Comment by Shabbatt** [comment]
how to run now to work?
how to specify the -n option to make it work?

**Comment by harold-b** [comment]
I’d need more info as no issue is produced with -n on our sode

**Reply by Shabbatt** [reply]
if you do not specify the -n option, it worked.
Once indicated, it crashes when it tests memory (warming up).
removed the parameter - above is an example - it still falls down. does not start.

this is what appears.

[Bladebit Disk Plotter]
  Heap size : 11.56 GiB ( 11840.27 MiB )
  Cache size : 200.00 GiB ( 204800.00 MiB )
  Bucket count : 64
  Alternating I/O: true
  F1 threads : 10
  FP threads : 10
  C threads : 10
  P2 threads : 10
  P3 threads : 10
  I/O threads: 1
  Temp1 block sz : 4096
  Temp2 block sz : 4096
  Temp1 path : I:\
  Temp2 path : I:\
  I/O metrics enabled.
  Allocating memory

hangs for a minute - and the window closes.

where to get the necessary data and send it to you?

I have 2 processors Е5-2697 v2 
48 streams are possible.
but I'm already trying different values to understand where the reason is.
memory 384gb. ddr3
tmpdir2=I:   this is a 1TB NVME drive

**Comment by Shabbatt** [comment]
timeout 30
"@echo off

set plots=1
set threads=48
set buckets=64
set tmpdir=M:\
set tmpdir2=I:\
set finaldir=H:\ChiaNew\
set poolkey=**************
set farmerkey=********************
echo %date% - %time%
echo -------------------------
bladebit.exe -w -t %threads% -f %farmerkey% -c %poolkey% diskplot -b %buckets% -a --cache 99G -t1 %tmpdir2% %finaldir% 
echo -------------------------
echo %date% - %time%
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
pause"	



[Bladebit Disk Plotter]
 Heap size      : 11.56 GiB ( 11840.27 MiB )
 Cache size     : 99.00 GiB ( 101376.00 MiB )
 Bucket count   : 64
 Alternating I/O: true
 F1  threads    : 48
 FP  threads    : 48
 C   threads    : 48
 P2  threads    : 48
 P3  threads    : 48
 I/O threads    : 1
 Temp1 block sz : 4096
 Temp2 block sz : 4096
 Temp1 path     : I:\
 Temp2 path     : I:\
 I/O metrices enabled.
 Allocating memory

After that, it hangs for a minute, and the window closes.

**Reply by Shabbatt** [reply]
important point. I repeat.
until I specified the -n option
the app worked. ploting was.
1 raft was created.
then I just specified this parameter, and after that, it behaves like this.
removed it.
but it still behaves like this.
that is the question.

madmax works.
I create a virtual disk in frames 2pcs.
one 110gb. the second 220gb. and completely in them ploting.
the finished raft is copied to the hard drive.
time takes 45 minutes.
memory runs at 1600
I think it can be faster. but you need to change the memory.
or the C drive, since it's a regular hard drive, and this application uses it a lot. 100% data exchange with it.

your application, I noticed the C drive doesn't use much.

**Reply by Shabbatt** [reply]
I create a virtual disk in frames 2pcs.
one 110gb. the second 220gb
RAM
